### PR TITLE
Adds CRUD endpoints and database operations

### PR DESCRIPTION
**Pull request by jayrbarr**

### CODE REVIEW

**Review by timothydadams — APPROVED** [review]
lgtm

File: client/App.jsx [hunk 1]
```diff
@@ -19,11 +19,7 @@ class App extends React.Component {
 
   componentDidMount() {
     axios
-      .get("/mainbody", {
-        params: {
-          _id: proxyId,
-        },
-      })
+      .get(`/mainbody/${proxyId}`)
```
Comment (JasperCDev) [inline client/App.jsx:22]: What was the point of this change?

**Review by JasperCDev — APPROVED** [review]
lgtm

File: client/App.jsx (hunk 1 above)
Comment (jayrbarr) [reply client/App.jsx:22]: I wanted the id in the ending not the parameters, and the whole object was messed up anyways. I didn't understand it. Now I do.

**Pull request merged**